src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs

```diff
@@ -130,8 +130,10 @@ internal partial class WinHttp
         public const uint WINHTTP_AUTH_TARGET_SERVER = 0x00000000;
         public const uint WINHTTP_AUTH_TARGET_PROXY = 0x00000001;
 
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="It is property descriptor, not secret value.")]
```

Comment:
I thought that it is the line with `password` needs to be suppressed.

Reply:
yep, this one is strange. The tool consider line 134 and 135 as one line for some reason.

Reply:
It is the same case for all your comments.

Reply:
I see. Thanks!